### ISSUE TITLE
Tabular format support?

### ISSUE THREAD

**Issue by RomeoDev777** [michael-brade/LaTeX.js#113]
Is it possible to put text in columns evenly?
like
 \begin{tabular}{ l | c | r }
  1 & 2 & 3 \\
  4 & 5 & 6 \\
  7 & 8 & 9 \\
\end{tabular}

**Comment by michael-brade** [comment]
Yeah, it will be possible but not yet :-p It will be one of the most interesting and fun things to implement!

**Comment by RomeoDev777** [comment]
Thanks!

**Comment by monaxiko** [comment]
Hi everyone,

Do you know when it will be available? Many thanks!

**Comment by michael-brade** [comment]
I don't know yet. I don't have much time at the moment. I hope to able to continue during the holidays a little.

**Comment by mkhan45** [comment]
Hey, I don't want to cause you any stress, but I've added a truth table generator to CalcuLaTeX and once LaTeX.js supports tabular they'll be able to render on https://calcula.tech so I would greatly appreciate it if you could add support soon

**Comment by jhpohovey** [comment]
Seeing as this is a rather stale issue, is it generally safe to assume that there are no near-term plans for adding this feature?

**Comment by ox223252** [comment]
Hello, can we hope to get tabular managed ? :)
